src/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -89,110 +78,14 @@ const AdminDashboard = () => {
-                                    </Grid>
-                                </Grid>
-                                <Divider />
-                                <ListItem className={classes.list}>
-                                    {details && (
-                                        <ListItemText
-                                            primary={`${
-                                                getSMSUsage() !== undefined
-                                                    ? getSMSUsage()
-                                                    : '?'
-                                            } SMS ${t('consumed')} ${t('on')} ${
-                                                getSMSQuota() !== undefined
-                                                    ? getSMSQuota()
-                                                    : '?'
-                                            } ${t('available')}`}
-                                        />
-                                    )}
-                                </ListItem>
-                            </Paper>
-
-                            <Box m={6} />
-
-                            <Paper elevation={0}>
-                                <Grid container alignItems="center" spacing={2}>
-                                    <Grid item>
-                                        <FingerprintIcon
-                                            className={classes.icon}
-                                        />
-                                    </Grid>
-                                    <Grid item>
-                                        <Typography variant="h6">
-                                            {t('api-identifier')}
-                                        </Typography>
-                                    </Grid>
-                                    <Grid item xs className={classes.textRight}>
-                                        <CopyToClipboard text={token}>
-                                            <Link
-                                                href="#"
-                                                onClick={preventDefault}
-                                                variant="body2">
-                                                {t('click-to-copy')}
-                                            </Link>
-                                        </CopyToClipboard>
-                                    </Grid>
-                                </Grid>
-                                <Divider />
-                                <ListItem className={classes.list}>
-                                    <ListItemText
-                                        className={classes.apiTokenListItem}
-                                        primary={token}
-                                    />
-                                </ListItem>
-                            </Paper>
+                            <UserDetails onRoomEdit={onRoomEdit} />
```

Comment:
UserDetails component with RoomEdit data in params? Consider another name or split component ?

Reply:
#458